Microsoft.Toolkit.Uwp.UI.Controls/TokenizingTextBox/TokenizingTextBox.cs

```diff
@@ -73,11 +73,26 @@ protected override void OnApplyTemplate()
             ContextFlyout = menuFlyout;
         }
 
-        private void AutoSuggestBox_KeyDown(object sender, KeyRoutedEventArgs e)
+        private async void AutoSuggestBox_KeyDown(object sender, KeyRoutedEventArgs e)
         {
             switch (e.Key)
             {
                 case VirtualKey.Back:
+                    {
+                        TextBox autoSuggestTextBox = _autoSuggestBox.FindDescendant<TextBox>() as TextBox;
```

Comment:
I'd suggest doing this in OnApplyTemplate and storing the result as walking the visual tree can be expensive, and the textbox shouldn't change out from underneath us vs. the user pressing backspace a bunch.